src/planned-tests.html

```diff
@@ -39,32 +39,9 @@ <h1 class="govuk-heading-xl">
         {{ pageTitle }}
       </h1>
 
-      <h2 class="govuk-heading-m govuk-!-margin-top-6 govuk-!-margin-bottom-2">
-        Tuesday 29 June 2021
-      </h2>
-      <h3 class="govuk-body govuk-!-margin-bottom-6">
-        Reading, Berkshire
-      </h3>
       <p class="govuk-body">
-        The UK government will send a test alert between 1pm and 2pm.
+        There are currently no planned tests of emergency alerts.
```

Comment:
This could maybe do with a link to the ‘past alerts’ page to help people who are looking for info about the Reading test.

Also not sure how I feel about saying ‘no planned tests’ while one is ongoing.

@karlchillmaid might have thoughts on this too

Reply:
Yeah, ‘no planned tests’ during a test feels a bit weird.

Could we do a thing where we say there is one, a link to the current alert? Or does that make things a bit fiddly?

Reply:
Linking to past alerts afterwards sounds good, though.

Reply:
Yeah, we can do that: we know what the URL will be.

Do you want to suggest some changes?